static/src/js/zustand/slices/__tests__/createEarthdataDownloadRedirectSlice.test.ts

```diff
@@ -0,0 +1,55 @@
+import useEdscStore from '../../useEdscStore'
+
+describe('createEarthdataDownloadRedirectSlice', () => {
+  test('sets the default state', () => {
+    const zustandState = useEdscStore.getState()
+    const { earthdataDownloadRedirect } = zustandState
+
+    expect(earthdataDownloadRedirect).toEqual({
+      redirect: '',
+      setRedirect: expect.any(Function)
+    })
+  })
+
+  describe('setRedirect', () => {
+    test('updates redirect', () => {
+      const zustandState = useEdscStore.getState()
+      const { earthdataDownloadRedirect } = zustandState
+      const { setRedirect } = earthdataDownloadRedirect
+      const newRedirect = 'earthdata-download://authCallback&token=abc123'
+
+      setRedirect(newRedirect)
+
+      const updatedState = useEdscStore.getState()
+      const { earthdataDownloadRedirect: updatedRedirect } = updatedState
+      expect(updatedRedirect.redirect).toBe(newRedirect)
+    })
+
+    test('can clear redirect by setting empty string', () => {
```

Comment:
This test, along with `can handle various redirect URL formats` aren't testing any additional logic performed by the setRedirect function, so they aren't providing any additional value to the test suite.

If setRedirect had any logic in it, conditionals or something like that, testing the various branches is great. But as is, this is just testing the same line in the code 3 times. Potentially adding more maintenance time in the future if something changes